tables/results/Table1-molecular-subtypes.tsv

```diff
@@ -64,4 +62,6 @@ Neuronal and mixed neuronal-glial tumor	GNT, KIAA1549-BRAF	2
 Neuronal and mixed neuronal-glial tumor	GNT, other MAPK	1
 Neuronal and mixed neuronal-glial tumor	GNT, other MAPK, FGFR	1
 Neuronal and mixed neuronal-glial tumor	GNT, RTK	2
-Tumors of sellar region	CRANIO, ADAM	27
+Other	CHDM, conventional	2
+Other	CHDM, poorly differentiated	4
```

Comment:
While we use Other for display tables, this should probably remain as `Chordoma` for this table.

Reply:
Hm, maybe I should change this to a whole new column that uses `broad_histology_display` unless it's "Other", in which case `broad_histology`. Then, the column will not match either of our internal variables `broad_histology<_display>`, so I feel it needs a new name altogether to avoid confusion with those (or is this more confusing?). Maybe something like `pbta_broad_histology`?